x/slashing/handler.go

```diff
@@ -42,6 +42,10 @@ func handleMsgUnjail(ctx sdk.Context, msg MsgUnjail, k Keeper) sdk.Result {
 		return ErrNoValidatorForAddress(k.codespace).Result()
 	}
 
+	if info.Tombstoned {
```

Comment:
This seems redundant with setting `JailedUntil` to `MaxTime`.

Reply:
(I'm fine with an extra field but would prefer to avoid the redundancy)

Reply:
Yeah, it somewhat is, but as described above, I need the `info.Tombstoned` to tell whether you're in jail for a byzantine fault or liveness fault.  So I could get away with not setting the `JailedUntil` to anything and just check if `info.Tombstoned == true` in the handler for `MsgUnjail` (which I do), but figured it is cleaner and more "accurate" to just set `JailedUntil` to `MaxTime`, for like UIs and whatnot.

Reply:
Ah I see, ok, that's fine.